index.d.ts

```diff
@@ -57,17 +57,17 @@ declare namespace Router {
     store: any
   ) => void;
 
-  interface ConstraintStrategy<V extends HTTPVersion> {
+  interface ConstraintStrategy<V extends HTTPVersion, T = string> {
     name: string,
     mustMatchWhenDerived?: boolean,
     storage() : {
-      get(version: String) : Handler<V> | null,
-      set(version: String, store: Handler<V>) : void,
-      del(version: String) : void,
+      get(version: T) : Handler<V> | null,
```

Comment:
Would you mind updating this variable to be named `value` or something less version specific? Not strictly a result of your change but it is a bit more confusing now that the type of this piece is generic

Reply:
Perhaps the name of the `store` variable should also be changed, as this has simply been taken over from the semver-store?

Reply:
Oh -- yep definitely, that's a `handler` not a `store`. Thanks for leaving it better than you found it!!